pcweb/pages/docs/getting_started/introduction.py

```diff
@@ -310,34 +369,41 @@ def intro2() -> rx.Component:
 @docpage()
 def introduction():
     return rx.box(
-        intro1(),
-        intro2(),
+        docheader("Reflex Docs"), 
```

Comment:
Personally I prefer the intro as it was before as it looked cleaner, it's a bit overwhelming now with the first paragraph.